.gitignore

```diff
@@ -1,13 +1,218 @@
-bin
-obj
-build
-_ReSharper.*
-*.csproj.user
-*.resharper.user
-*.ReSharper.user
-*.ReSharper
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
This is a significant change to the gitignore. Is it necessary?

Reply:
Just using the standard .gitignore for .Net projects provided by VS2017 - same for the .gitattributes. I can remove them if you want - although, I noticed another copy fail on this file...

Reply:
If it's VS provided, and will make future contributions easier, we can leave it.